xwiki-commons-core/xwiki-commons-configuration/xwiki-commons-configuration-api/pom.xml

```diff
@@ -44,10 +44,21 @@
       <artifactId>xwiki-commons-component-api</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.xwiki.commons</groupId>
+      <artifactId>xwiki-commons-context</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
The alternative was to put the code in ``xwiki-commons-context`` and add there the dependency on ``xwiki-commons-configuration-api``, but @tmortagne noted that it's more probable for a module that requires the configuration API to also require the context API, than the other way.